app/views/recipes/_recipe_card.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='recipe'>
+  <div>
+    <%= link_to recipe_path(recipe) do %>
+      <h2><%= recipe.name %></h2>
+    <% end %> 
+    <%= button_to 'Remove', recipe_url(recipe), method: :delete %>
+  </div>
+  <p><%= recipe.description %></p>
+</div>
```

Comment:
- [ ] Good job so far! It looks like the recipe description is not appearing even though you have added it here. I think the problem is in the recipe creation form so please check if the input for adding the recipe description is linked to the `description` attribute. According to the wireframe, here is the comparison currently 👇 
**Your web**
![image](https://user-images.githubusercontent.com/90222110/218268307-d652e10b-597a-424b-9626-9c9da3cdd10f.png)
**Wireframe**
![image](https://user-images.githubusercontent.com/90222110/218268332-3034c418-ca2b-4d03-9fe6-94cc939f36a6.png)